starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,86 @@
+/*Some necessary adjustments*/
+h1, body, html, main {
+    margin: 0;
+  }
+/*Import the font and set the default size*/
+html {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 16px
+}
+/* ---- reset ---- */
+
+hr {
+    border: 1px solid #ccc;
+}
+
+ul {
+    padding: 0;
+    list-style: none;
+}
+
+/* ---- typography ---- */
+
+body {
+    color: #444;
+}
+h1 {
+    color: #fff;
+    font-size: 100px;
+    font-weight: normal;
+}
+h2{
+    font-weight: normal;
+    font-size: 28px;
+    color: #000;
+ }
+/* ---- layout ---- */
+
+#cover {
+    margin: 0 auto;
+    height: 100vh;
+    max-height: 800px;
+    max-width: 960px;
+    background: url(../images/apple-pie.jpg);
+    background-size: cover;
+    background-position: center;
+}
+h1 {
+    position: relative;
+    float: left;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Well done. If you want to learn more about centering in css, here you have an amazing guide: https://css-tricks.com/centering-css-complete-guide/